Sources/Core/AWSSDKChecksums/Sources/AWSSDKChecksums/AWSChecksumCalculationMode.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+public enum AWSChecksumCalculationMode: String {
```

Comment:
I like this idea of whenRequired vs whenSupported

Reply:
This addition to config is part of the V2 SEP 👍